File: feitos/calculosimples.py
string1 = input()
string2 = input()

v1 = string1.split(" ")
v2 = string2.split(" ")

np1 = int(v1[1])
vu1 = float(v1[2])

np2 = int(v2[1])
vu2 = float(v2[2])

total = (np1 * vu1) + (np2 * vu2)

print("VALOR A PAGAR: R$ %.2f" % total)

File: feitos/calculosimples.cpp
#include <iostream>

using namespace std;

int main(){
	int cp1, np1, cp2, np2;
	double vu1, vu2, valor;
	
	cin >> cp1 >> np1 >> vu1;
	cin >> cp2 >> np2 >> vu2;
	
	std::cout.precision(2);
	std:cout.setf(std::ios::fixed, std::ios::floatfield);
	
	valor = (np1 * vu1) + (np2 * vu2);
	
	cout << "VALOR A PAGAR: R$ " << valor << endl;
	
	return 0;
}
